projects/ion/src/lib/smart-table/smart-table.component.spec.ts

```diff
@@ -221,6 +222,61 @@ describe('IonSmartTableComponent', () => {
   });
 });
 
+describe('Table > columns header with tooltip', () => {
+  let columnHead: HTMLElement;
+  const columnsWithTooltip: Column[] = [
+    {
+      key: 'name',
+      label: 'Nome',
+      sort: true,
+      configTooltip: {
+        ionTooltipTitle: 'Eu sou um tooltip',
+      },
+    },
+    {
+      key: 'height',
+      label: 'Altura',
+      sort: true,
+    },
+  ];
+
+  const propsColumnWithTooltip: IonSmartTableProps<Character> = {
+    config: {
+      data,
+      columns: columnsWithTooltip,
+      pagination: {
+        total: 32,
+        itemsPerPage: 10,
+        page: 1,
+      },
+      loading: false,
+    },
+    events: {
+      emit: events,
+    } as SafeAny,
+  };
+
+  beforeEach(async () => {
+    await sut(propsColumnWithTooltip);
+  });
+
+  afterEach(() => {
+    fireEvent.mouseLeave(columnHead);
+  });
+
+  it('should render tooltip when it have a configTooltip', () => {
+    columnHead = screen.getByTestId('th-span-' + columnsWithTooltip[0].key);
+    fireEvent.mouseEnter(columnHead);
+    expect(screen.queryByTestId('ion-tooltip')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByTestId('ion-tooltip')).toBeVisible();
```
Maybe this is a more assertive way to test this, it will check if it have only one tooltip open (because of getByTestId) and will check if the tooltip is visible to user (it could be rendered, but not visible).